assets/pages/contributions-landing/contributionsLanding.jsx

```diff
@@ -28,10 +28,21 @@ import ContributionAwarePaymentLogosContainer from './containers/contributionAwa
 import { createPageReducerFor } from './contributionsLandingReducer';
 
 
+// ----- Redux Store ----- //
+
+const countryGroupId: CountryGroupId = detect();
+
+const store = pageInit(createPageReducerFor(countryGroupId));
+
+const oneOffOnlyVariant = (store && store.getState().common.abParticipations.ContributeLandingOneOffOnlyTest) || 'notintest';
```

Comment:
I don't think you need to check the store exists here. I believe the type says that it does exist, and in the event that it doesn't we have bigger problems than failing to look up the test state 😯.

Reply:
You do, othewise you get this:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ flow-typed/npm/redux_v3.x.x.js:56:87

S [1] is incompatible with S [1].

 [1] 30│   declare export type Reducer<S, A> = (state: S | void, action: A) => S;
       :
     53│   declare export function bindActionCreators<A, C: ActionCreator<A, any>, D: DispatchAPI<A>>(actionCreator: C, dispatch: D): C;
     54│   declare export function bindActionCreators<A, K, C: ActionCreators<K, A>, D: DispatchAPI<A>>(actionCreators: C, dispatch: D): C;
     55│
     56│   declare export function combineReducers<O: Object, A>(reducers: O): CombinedReducer<$ObjMap<O, <S>(r: Reducer<S, any>) => S>, A>;
     57│
     58│   declare export var compose: $Compose;
     59│ }


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ assets/helpers/page/page.js:170:29

null [1] is incompatible with Reducer [2].

     assets/helpers/page/page.js
 [1] 149│   pageReducer: Reducer<S, A> | null = null,
        :
     167│   const commonReducer = createCommonReducer(initialState);
     168│
     169│   return createStore(
     170│     combineReducers({ page: pageReducer, common: commonReducer }),
     171│     storeEnhancer(thunk),
     172│   );
     173│ }

     flow-typed/npm/redux_v3.x.x.js
 [2]  56│   declare export function combineReducers<O: Object, A>(reducers: O): CombinedReducer<$ObjMap<O, <S>(r: Reducer<S, any>) => S>, A>;



Found 2 errors
```

Reply:
Hmm, that seems to be a problem with the way I've annotated the page `init` function. Ok, not an issue for this PR, looks good.